main.js

```diff
@@ -25,7 +15,7 @@ var activitySeconds = document.querySelector('.activity-seconds');
 var startActivityButton = document.querySelector('.start-activity');
 var startStopButton = document.querySelector('.start-stop-button');
 
-//var activityErrorMessage = document.querySelector('.activity-error');
+//var activityErrorMessage = document.querySelector('.activity-error'); delete?
```

Comment:
Do we need this line? or no? just wondering why it's commented out.

Reply:
Nice catch. I think we can definitely delete this since it's redundant with the querySelector on line 20. Both are targeting '.activity-error'. Must be leftover from older code.